lms/djangoapps/grades/docs/how_tos/how-to-regrade-everyone-in-a-course.rst

```diff
@@ -8,27 +8,18 @@ grades for every learner in a course.
 Steps to trigger course-wide re-grade
 =====================================
 
-1. Go to the Studio grading settings for the course,
-   for example https://studio.stage.edx.org/settings/grading/edX/DemoX/Demo_Course .
+1. Go to LMS Django Admin and add an entry in ComputeGradesSetting, in the Grades section. 
+   ``/admin/grades/computegradessetting/``
+   (this requires the persistent_grades_admin role in app-permissions)
 
-2. Make a trivial edit to one of the assignment type names.  For example,
-   for an assignment type named "Homework", add a space to get "Homework ",
-   and then delete the space.
+2. Put the course key of the course you wish to recompute into the ``course ids`` field.
+   You can enter multiple whitespace-separated keys. Save the model.
 
-3. Studio should now alert you that you've made changes;
-   click the "Save Changes" button at the bottom of your screen.
+3. Go to tools-edx-jenkins and navigate to Grading/(prod,stage)-edx-compute_grades
```

Comment:
wasn't as sure how to do this one, is this okay?

Reply:
Yeah, that looks fine.